doc/articles/uno-development/troubleshooting-memory-issues.md

```diff
@@ -58,12 +59,12 @@ private void EnableViewsMemoryStatistics()
 	);
 }
 ```
-You'll also need to add the following logger filter:
-```
+  You'll also need to add the following logger filter:
+```csharp
 builder.AddFilter("Uno.UI.DataBinding.BinderReferenceHolder", LogLevel.Information );
 ```
-As well as this package NuGet (you will need to update to the latest Uno.UI nuget version):
-```
+  As well as this package NuGet (you will need to update to the latest Uno.UI nuget version):
+```xaml
 <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.0.13" />
```

Comment:
This should be enough for dependencies. Which methods are missing otherwise if you don't include Uno.Core ?

Reply:
That might be the case for the example in question.  I'll be able to confirm later today.  

FWIW, I came across this issue when trying to access `public static ILogger Log(this Type forType)` in a Uno Class Library project.  There, I'm pretty sure (again, I'll confirm later today) that I tried using `Uno.UI.Adapter.Microsoft.Extensions.Logging` first without success.

Reply:
Could be I was using the wrong version?

Reply:
OK, for the purposes of this issue, `Uno.UI.Adapter.Microsoft.Extensions.Logging` contains the extensions methods in question.  Thanks for pointing that out.  I did not as I was updating an Uno3 project to Uno4.

FWIW, in the Uno class library project that sent me down this path, I'm also using `!url.IsNullOrWhiteSpace()` which is in `Uno.Core` but not ``Uno.UI.Adapter.Microsoft.Extensions.Logging`.

Reply:
Correct, this extension method is not referenced there, it's in now in https://nuget.info/packages/Uno.Core.Extensions/4.0.1. We wanted to remove the dependency for dependents of Uno, but you're welcome to use it :)